src/components/Tooltip/Tooltip.tsx

```diff
@@ -23,17 +23,29 @@ const Tooltip = forwardRef(
       [isDescription]
     );
 
+    // Modified tooltipSetInstance to call both setInstance and updateInstance
+    const setTooltipInstance = useCallback(
+      (popoverInstance: PopoverInstance | undefined) => {
+        if (otherProps?.setInstance) {
+          otherProps.setInstance(popoverInstance);
+        }
```

Comment:
I do not think we need the extra `useCallback` here.

If you rename `updateInstance` to `setInstance` then you do it there as well

also you can make it in a single line:
```suggestion
        otherProps?.setInstance?.(popoverInstance);
```